File: src/main/java/pers/lyks/spring/example/service/impl/EmployeeServiceImpl.java
package pers.lyks.spring.example.service.impl;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import pers.lyks.spring.example.dao.EmployeeDao;
import pers.lyks.spring.example.model.EmployeeModel;
import pers.lyks.spring.example.service.EmployeeService;

/**
 * @author lawyerance
 * @version 1.0 2018-08-04
 */
@Service
public class EmployeeServiceImpl implements EmployeeService {
    @Autowired
    private EmployeeDao employeeDao;

    @Override
    public EmployeeModel get(long id) {
        return employeeDao.get(id);
    }
}

File: src/main/kotlin/pers/lyks/spring/example/service/impl/EmployeeServiceImpl.kt
package pers.lyks.spring.example.service.impl

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service
import pers.lyks.spring.example.mapper.EmployeeMapper
import pers.lyks.spring.example.model.EmployeeModel
import pers.lyks.spring.example.service.EmployeeService

import javax.annotation.Resource

/**
 * @author lawyerance
 * @version 1.0 2018-08-04
 */
@Service
class EmployeeServiceImpl : EmployeeService {
    @Resource
    private val employeeMapper: EmployeeMapper? = null

    override fun get(id: Long): EmployeeModel {
        return employeeMapper!![id]
    }
}
